docs/public/_redirects

```diff
@@ -299,52 +299,60 @@ https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 /api/no-ssr/ /base/api/no-ssr/ 301
 /pt/api/no-ssr/ /pt/base/api/no-ssr/ 301
 /zh/api/no-ssr/ /zh/base/api/no-ssr/ 301
-/api/form-control-unstyled/ /base/api/form-control-unstyled/ 301
-/pt/api/form-control-unstyled/ /pt/base/api/form-control-unstyled/ 301
-/zh/api/form-control-unstyled/ /zh/base/api/form-control-unstyled/ 301
-/api/badge-unstyled/ /base/api/badge-unstyled/ 301
-/pt/api/badge-unstyled/ /pt/base/api/badge-unstyled/ 301
-/zh/api/badge-unstyled/ /zh/base/api/badge-unstyled/ 301
+/api/form-control-unstyled/ /base/react-form-control/components-api/#form-control-unstyled 301
+/pt/api/form-control-unstyled/ /pt/base/react-form-control/components-api/#form-control-unstyled 301
+/zh/api/form-control-unstyled/ /zh/base/react-form-control/components-api/#form-control-unstyled 301
+/api/badge-unstyled/ /base/react-badge/components-api/#badge-unstyled 301
+/pt/api/badge-unstyled/ /pt/base/react-badge/components-api/#badge-unstyled 301
+/zh/api/badge-unstyled/ /zh/base/react-badge/components-api/#badge-unstyled 301
 /api/backdrop-unstyled/ /base/react-modal/ 301
 /pt/api/backdrop-unstyled/ /pt/base/react-modal/ 301
 /zh/api/backdrop-unstyled/ /zh/base/react-modal/ 301
-/api/switch-unstyled/ /base/api/switch-unstyled/ 301
-/pt/api/switch-unstyled/ /pt/base/api/switch-unstyled/ 301
-/zh/api/switch-unstyled/ /zh/base/api/switch-unstyled/ 301
-/api/slider-unstyled/ /base/api/slider-unstyled/ 301
-/pt/api/slider-unstyled/ /pt/base/api/slider-unstyled/ 301
-/zh/api/slider-unstyled/ /zh/base/api/slider-unstyled/ 301
-/api/button-unstyled/ /base/api/button-unstyled/ 301
-/pt/api/button-unstyled/ /pt/base/api/button-unstyled/ 301
-/zh/api/button-unstyled/ /zh/base/api/button-unstyled/ 301
+/api/switch-unstyled/ /base/react-switch/components-api/#switch-unstyled 301
+/pt/api/switch-unstyled/ /pt/base/react-switch/components-api/#switch-unstyled 301
+/zh/api/switch-unstyled/ /zh/base/react-switch/components-api/#switch-unstyled 301
+/api/slider-unstyled/ /base/react-slider/components-api/#slider-unstyled 301
+/pt/api/slider-unstyled/ /pt/base/react-slider/components-api/#slider-unstyled 301
+/zh/api/slider-unstyled/ /zh/base/react-slider/components-api/#slider-unstyled 301
+/api/button-unstyled/ /base/react-button/components-api/#button-unstyled 301
+/pt/api/button-unstyled/ /pt/base/react-button/components-api/#button-unstyled 301
+/zh/api/button-unstyled/ /zh/base/react-button/components-api/#button-unstyled 301
 /api/textarea-autosize/ /base/api/textarea-autosize/ 301
 /pt/api/textarea-autosize/ /pt/base/api/textarea-autosize/ 301
 /zh/api/textarea-autosize/ /zh/base/api/textarea-autosize/ 301
-/api/modal-unstyled/ /base/api/modal-unstyled/ 301
-/pt/api/modal-unstyled/ /pt/base/api/modal-unstyled/ 301
-/zh/api/modal-unstyled/ /zh/base/api/modal-unstyled/ 301
-/api/option-group-unstyled/ /base/api/option-group-unstyled/ 301
-/:lang/api/option-group-unstyled/ /:lang/base/api/option-group-unstyled/ 301
```

Comment:
Using `/:lang/` also matches urls starting with `/base/`, which causes links like: https://mui.com/base/api/tabs-list-unstyled/ to be broken.